doc/source/user_guide/example_data.rst

```diff
@@ -3,17 +3,70 @@
 Installing the Example data
 ===========================
 
-The examples scripts in the user guide as well as the pySTEPS build-in tests
+The examples scripts in the user guide, as well as the pySTEPS build-in tests,
 use the example radar data available in a separate repository:
 `pysteps-data <https://github.com/pySTEPS/pysteps-data>`_.
 
-The data must be downloaded manually into your computer and the ref:pystepsrc"
-file need to configured to point to that example data.
+The easiest way to install the example data is by using the
+:func:`~pysteps.datasets.download_pysteps_data` and
+:func:`~pysteps.datasets.create_default_pystepsrc` functions from
+the :mod:`pysteps.datasets` module.
+
+Installation using the datasets module
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below is a snippet code that can be used to install can configure `pystepsrc` file to
+point to that example data.
+
+
+In the example below, the example data is placed in the user's home folder under the
+**pysteps_data** directory. It also creates a default configuration file that points to
+the downloaded data and places it in the $HOME/.pysteps (Unix and Mac OS X) or
+$USERPROFILE/pysteps (Windows). This is one of the default locations where pysteps
+looks for the configuration file (see :ref:`pysteps_lookup` for
+more information).
+
+.. code-block:: python
+
+    import os
+
+    # Import the helper functions
+    from pysteps.datasets import download_pysteps_data, create_default_pystepsrc
+
+    # In this example we will place it in the user's home folder on the
+    # `pysteps_data` folder.
+    home_dir = os.path.expanduser("~")
+    pysteps_data_dir_path = os.path.join(home_dir, "pysteps_data")
+
+    # Download the pysteps data.
+    download_pysteps_data(pysteps_data_dir_path, force=True)
+
+    # Create a default configuration file that points to the downloaded data.
+    # By default it will place the configuration file in the
+    # $HOME/.pysteps (unix and Mac OS X) or $USERPROFILE/pysteps (windows).
+    config_file_path = create_default_pystepsrc(pysteps_data_dir_path)
+
+Note that for these changes to take effect you need to restart the python interpreter or
+use the :func:`pysteps.load_config_file` function as follows::
+
+    # Load the new configuration file and replace the default configuration
+    import pysteps
+    pysteps.load_config_file(path_to_my_config_file, verbose=True)
```

Comment:
Just a detail: using `config_file_path` as argument here instead of `path_to_my_config_file` would make a direct and logic link to the code above (line 47)

Reply:
Solved in 77cd9c8.